configuration/src/main/java/com/optum/sourcehawk/configuration/SourcehawkConfiguration.java

```diff
@@ -18,6 +19,7 @@
     /**
      * The remote files to inherit from in URL form
      */
+    @JsonIgnore
```

Comment:
When flattening, without this, it will include the remove links from files recursively.  With this in place it will not include this.  I see this as useful for when you want to actually use the output WITHOUT the remote links in it.

Reply:
Okay, let's add a comment explaining that.